packages/protocol/foundry.toml

```diff
@@ -18,6 +18,7 @@ remappings = [
   "openzeppelin/contracts/=node_modules/@openzeppelin/contracts/",
   "@risc0/contracts/=node_modules/risc0-ethereum/contracts/src/",
   "@solady/=node_modules/solady/",
+  "solady/=node_modules/solady/",
```

Comment:
please notice the previous line.

Reply:
@dantaik They are different. I can't compile successfully. Can you?